enhancements/network/ovn-kubernetes-localnet-api.md

```diff
@@ -0,0 +1,557 @@
+        - localnet: physnet  <---- has to match the NAD config.spec.name. OR 
+                                   the NAD config.spec.physicalNetworkName
+          bridge: br-ex <--------- OVS switch
+```
+* To overcome this, and avoid exposing the net-conf-network-name in the CUDN CRD spec a new field should be introduced.
+The new field should allow users pointing to the bridge-mapping network-name they defined in the node.
+The field should be translated to the CNI physicalNetworkName field.
+
+#### MTU
+By default OVN-K sets the pod interface with MTU 1400, unless otherwise specified.
+
+For localnet topology this is not optimal because the host physical network-interface usually has default MTU 1500.
+In case no MTU is specified the pod interface connected to the host localnet is configured with MTU 1400 resulting 
+in bad performance and bad UX requiring troubleshooting the MTU alignment across the stack.
+
+As long OVN-K does not set default MTU (1500) for localnet topology network, 
+the controller should specify MTU to be 1500 explicitly in the NAD, if no MTU was specified in the CUDN spec.
+
+### Workflow Description
+
+The CUDN CRD controller should be changes accordingly to support localnet topology.
+It should validate localnet topology configuration and generate corresponding NADs for localnet as other topologies (Layer2 & Layer3)
+in the selected namespaces.
+
+As of today the CUDN CRD controller does not allow mutating `spec.network`.
+
+OVN-Kubernetes support NAD spec mutations on day 2.
+For the new the spec to take place, restarting the workloads is necessary.
+
+In order to comply with NAD spec mutation support and provide better UX,
+the CUDN CRD controller should allow `spec.network` mutations for localnet topologies.
+Enable changing at least the following fields: `MTU`, `VLAN`, `excludeSubnets` and `physicalNetworkName`.
+
+In any order:
+- The user configures localnet bridge mapping on the nodes, e.g.: using NNCP.
+- Create CUDN, specifying the bridge-mapping network name in the spec.
+
+#### Generating the NAD
+##### OVS bridge-mapping’s network-name
+Introduce attribute to point ovs-bridge bridge-mapping network-name.
+This attribute name should be translated to the CNI “physicalNetworkName” attribute.
+
+Proposal the CUDN spec field name:  “physicalNetworkName”.
+
+Alternatively, the field name could be `name`.
+In context of the localnet topology configuration it makes sense. For example: 
+```yaml
+spec:
+  network:
+    topology: Localnet
+    localnet:
+      name: "tenantblue"
+```
+
+##### MTU
+Should be translated to the CNI “mtu” attribute.
+If not specified the controller should specify default MTU (1500) in the NAD spec.config.
+
+##### VLAN
+Should be translated to the CNI “vlanID” attribute.
+If not specified it should not be present in NAD spec.config.
+
+##### Subnets
+The subnets and exclude-subnets should be in CIDR form, similar to Layer2 topology subnets.
+
+##### Persistent IPs
```

Comment:
I miss explicitly indicating which is the default. Which I assume to be false.

Reply:
Yes default is false. Added text to clarify that.